src/test/java/org/folio/rest/impl/CustomFieldsImplTest.java

```diff
@@ -96,6 +96,14 @@ public void postTextboxCustomFieldWithEmailFormat() throws IOException, URISynta
     assertEquals(TextField.FieldFormat.EMAIL, customField.getTextField().getFieldFormat());
   }
 
+  @Test
+  public void postDisplayInAccordion() throws IOException, URISyntaxException {
+    var customFieldJson = readFile("fields/post/textbox/postDisplayInAccordion.json");
+    CustomField customField = createCustomField(customFieldJson);
+    assertNotNull(customField.getTextField());
+    assertEquals(TextField.FieldFormat.EMAIL, customField.getTextField().getFieldFormat());
```

Comment:
The test case should assert on the `displayInAccordion` field instead of `textField` and `fieldFormat`.